src/object_store/wrapped.rs

```diff
@@ -42,6 +42,9 @@ impl InternalObjectStore {
             schema::ObjectStore::S3(S3 { bucket, .. }) => {
                 Url::from_str(&format!("s3://{bucket}")).unwrap()
             }
+            schema::ObjectStore::GCS(GCS { bucket, .. }) => {
+                Url::from_str(&format!("s3://{bucket}")).unwrap()
```

Comment:
Sure this shouldn't be `gcs://`?

Reply:
It looks like according to [object_store](https://github.com/apache/arrow-rs/blob/eb5ac6953a9c24b8c5d03c95e4c49128b4e3b696/object_store/src/gcp/mod.rs#L942) the scheme should be `gs://`, so unless objections intending to go that route

Reply:
Changed it to `gs://`. Marking resolved.